components/EditionMenu.vue

```diff
@@ -26,6 +26,17 @@
         currAlignment.value = Object.keys(textAlign)[currIndex.value];
         emits('changeTextAlign', currAlignment.value);
     }
+
+    function sendUp() {
+        emits('frontEvent');
+        generateToast('info', 'Element moved forward.');
+    }
+
+    function sendDown() {
+        emits('backEvent');
+        generateToast('info', 'Element moved backward.');
```

Comment:
I changed the text again as requested in my Pr. it is now:

Element moved forward one step.
Element moved backward one step.

to make it crystal clear to the user ^^

Reply:
ok